.github/workflows/check.yml

```diff
@@ -6,12 +6,25 @@ jobs:
   check:
     strategy:
       matrix:
-        runs-on: [ubuntu-latest, macos-latest]
-    runs-on: ${{ matrix.runs-on }}
+        runs-on: [ '{"runs-on": "ubuntu-latest"}', 
+          '{"runs-on": "macos-latest"}', 
+          '{"runs-on": "ubuntu-latest", "container": "ubuntu:18.04"}' ]
+    runs-on: ${{ fromJSON(matrix.runs-on).runs-on }}
+    container: ${{ fromJSON(matrix.runs-on).container }} 
 
     steps:
     - uses: actions/checkout@v2
 
+    - uses: actions/setup-node@v1
```

Comment:
out of curiosity, why does this version of Ubuntu have a special case?

Reply:
Its a container. The check is ensuring that the action is working on containers aswell